src/pages/profile.tsx

```diff
@@ -373,13 +373,15 @@ const Profile: NextPage = () => {
                           type="number"
                           {...register("seatAvail", { valueAsNumber: true })}
                         />
-                        <Note>
-                          Registering 0 available seats will remove you from the
-                          app&apos;s recommendaiton generation.
-                        </Note>
                       </div>
                     )}
                   </div>
+                  {watch("role") == Role.DRIVER && (
```

Comment:
Could you explain why this has been moved outside of that parent div? they should all be conditionally rendered via the already existing watch on line 360, im confused as to why you have moved it outside. I think this is also working fine in my PR.

Reply:
Yes, before these changes, this is what this note looked like:
<img width="976" alt="Screenshot 2023-07-05 at 7 05 51 AM" src="https://github.com/sandboxnu/nucarpool/assets/39184335/352f826e-fc5c-4194-b6e5-61c584cb34fb">
imo, this looks weird because the buttons and the box aren't lining up. I fixed this by moving the note out of its previous position and adding this conditional rendering statement once again outside. So that it looks like this: 
<img width="759" alt="Screenshot 2023-07-05 at 7 06 11 AM" src="https://github.com/sandboxnu/nucarpool/assets/39184335/acaf4439-9add-4a45-a4c1-b8616cdef74a">

I think the second one looks a lot cleaner, but I can move it back if people think the primary one is better.

Reply:
And if there's a better way to do the same thing, just lmk, I just did it as a quick fix

Reply:
Oh okay yeah that makes sense and looks pretty good. On my PR i was working with the positioning of the Seat Availability field and so I ended up with it looking like below. I think yours actually looks better but someone will just have to handle the small merge conflicts. Also, if we choose to keep yours, just make sure to fix the typo for 'recommendation' in the note.

<img width="616" alt="Screenshot 2023-07-05 at 7 55 18 AM" src="https://github.com/sandboxnu/nucarpool/assets/93484080/d39944a2-54e8-4c80-8cb6-6f46cb6604f8">

Reply:
Sounds good